src/net/rpc.rs

```diff
@@ -2113,6 +2174,7 @@ impl ConversationHttp {
         handler_opts: &RPCHandlerArgs,
     ) -> Result<Option<StacksMessageType>, net_error> {
         let mut reply = self.connection.make_relay_handle(self.conn_id)?;
+        let burn_tip = SortitionDB::get_canonical_burn_chain_tip(sortdb.conn())?;
```

Comment:
Instead of doing database I/O on this hot path, can you just instead use `network.burnchain_tip` for the burnchain tip?  Thanks!